independent-publisher-connectors/NASA Image and Video Library/apiDefinition.swagger.json

```diff
@@ -0,0 +1,597 @@
+                                  "type": "string",
+                                  "description": "Format of media"
+                                }
+                              }
+                            },
+                            "description": "links"
+                          }
+                        }
+                      },
+                      "description": "items"
+                    },
+                    "links": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "href": {
+                            "type": "string",
+                            "description": "URL of result set"
+                          },
+                          "prompt": {
+                            "type": "string",
+                            "description": "Direction of result set"
+                          },
+                          "rel": {
+                            "type": "string",
+                            "description": "Link relation"
+                          }
+                        }
+                      },
+                      "description": "Links to additional result sets"
+                    },
+                    "metadata": {
+                      "type": "object",
+                      "properties": {
+                        "total_hits": {
+                          "type": "integer",
+                          "format": "int32",
+                          "description": "Total search hits"
+                        }
+                      },
+                      "description": "Metadata about search results"
+                    },
+                    "version": {
+                      "type": "string",
+                      "description": "version"
+                    }
+                  },
+                  "description": "collection"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Search media",
+        "description": "Search media",
+        "operationId": "Search",
+        "parameters": [
+          {
+            "name": "q",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Free text search terms to compare to all indexed metadata.",
+            "x-ms-summary": "Search term",
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "center",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "NASA center which published the media.",
+            "x-ms-summary": "NASA center"
+          },
+          {
+            "name": "description",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Terms to search for in \u201cDescription\u201d fields.",

```

Comment:
@paulculmsee Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in the English language.

Applies the same comment to all the unicodes in the connector artifacts.

Reply:
done